packages/web-twig/src/Resources/components/Header/stories/HeaderInvertedWithActionsAndDropdown.twig

```diff
@@ -0,0 +1,92 @@
+<Header color="inverted">
+    {% include '@components/Header/stories/_partials/SpiritLogo.twig' %}
+
+    <HeaderMobileActions dialogId="header_dialog_example_3_main_navigation">
+        <DropdownWrapper>
+            <Button
+                color="inverted"
+                data-spirit-toggle="dropdown"
+                data-spirit-target="#dropdownUserMobile"
+                isSquare
+            >
+                <Icon name="profile" />
+                <VisuallyHidden>My Profile</VisuallyHidden>
+            </Button>
+            <Dropdown id="dropdownUserMobile" placement="bottom-right">
+                <Stack hasSpacing>
+                    <a href="#">Dashboard</a>
+                    <a href="#">Profile</a>
+                    <a href="#">Favourites</a>
+                    <a href="#">Notifications</a>
+                    <a href="#">Sign out</a>
+                </Stack>
+            </Dropdown>
+        </DropdownWrapper>
+    </HeaderMobileActions>
+
+    <HeaderDesktopActions aria-label="Main navigation">
+        <HeaderNav>
+            <HeaderNavItem>
+                <HeaderLink href="#" isCurrent>Job offers</HeaderLink>
+            </HeaderNavItem>
+            <HeaderNavItem>
+                <HeaderLink href="#">Part-time jobs</HeaderLink>
+            </HeaderNavItem>
+            <HeaderNavItem>
+                <HeaderLink href="#">Inspiration</HeaderLink>
+            </HeaderNavItem>
+            <HeaderNavItem>
+                <HeaderLink href="#">Replies</HeaderLink>
+            </HeaderNavItem>
+            <HeaderNavItem>
+                <HeaderLink href="#">Employers</HeaderLink>
+            </HeaderNavItem>
+        </HeaderNav>
+    </HeaderDesktopActions>
+
+    <HeaderDesktopActions color="secondary">
+        <HeaderNav>
+            <HeaderNavItem UNSAFE_style="position: relative;">
```

Comment:
Maybe a foolish idea, but what about to apply `display: contents;` to the `DropdownWrapper`?
Then we could also use this component in the cases like this.

Reply:
I'm sorry, it's not true. It would affect the Dropdown menu. 🙈 
But maybe `flex`… 👀

Reply:
`display: contents` would make `DropdownWrapper` disappear from DOM which would lead to a misplaced `Dropdown`…

Originally, I was even considering rather a helper class like `position-relative` because that's the only thing I want it to do. Anything else could potentially cause more harm than good, I think.

Reply:
There is a high chance that with Breeze version we are able to solve this case without additional styles – @adamkudrna will take a look at it if there will be a task in sprint. 🙌🏻